sdk/daml-lf/validation/BUILD.bazel

```
@@ -126,6 +126,7 @@ da_scala_library(
         [
             "//canton:community_admin-api_proto_scala",
         ],
+    override_scalacopts = ["-Wconf:cat=deprecation&msg=in\\ proto\\ file:s"],
```

```suggestion
    scalacopts = ["-Wconf:cat=deprecation&msg=in\\ proto\\ file:s"],
```

You should not override the scalacopts but add to them.